tests/test_catalog.py

```diff
@@ -468,13 +499,56 @@ def test_catalog_list(
         json=api_resp,
     )
 
-    config.update_verbosity(verbose)
-    catalog.catalog_list(config)
+    catalog.catalog_list(config, output)
 
     captured = capsys.readouterr()
 
-    result = captured.out.strip().split("\n")
-    assert result == expected
+    update_golden = os.getenv("COMMODORE_GEN_GOLDEN", "False").lower() in (
+        "true",
+        "1",
+        "t",
+    )
+
+    result = captured.out
+
+    test_file = os.path.realpath(__file__)
+    test_directory = os.path.dirname(test_file)
+    golden_file = Path(test_directory) / "testdata" / "catalog_list" / expected
```

Comment:
Instead of mixing `os.path` and `pathlib.Path`, I'd do:

```suggestion
    golden_file = Path(__file__).absolute().parent / "testdata" / "catalog_list" / expected
```

Or `resolve()` to actually get `realpath()`, although it shouldn't matter (we do `.absolute()` elsewhere in the tests):

```suggestion
    golden_file = Path(__file__).resolve().parent / "testdata" / "catalog_list" / expected
```